### PR TITLE
show pages added

### PR DESCRIPTION
**Pull request by brovani**

### CODE REVIEW

File: app/controllers/spaces_controller.rb [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+    @spaces = Space.all
   end
 
   def new
     @space = Space.new
   end
 
+  # added by Bruno
+  def show
+    @space = Space.find(params[:id])
```
Comment (benediktms) [inline app/controllers/spaces_controller.rb:14]: remove

File: app/controllers/spaces_controller.rb [hunk 2]
```diff
@@ -17,9 +23,15 @@ def create
     end
   end
 
+  def destroy
+    @space = Space.find(params[:id])
```
Comment (benediktms) [inline app/controllers/spaces_controller.rb:27]: remove

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -8,7 +8,12 @@
     <div class="col">
 
       <% if user_signed_in? %>
-        <p>Welcome back</p>
+      <%# Bruno added 4 lines below%>
+        <p>Welcome back [insert user name] </p>
+      <%= link_to 'Click here to view all our spaces and make a new booking', spaces_path %>
+      <br>
+      <%= link_to 'View all your existing bookings', bookings_path %>
```
Comment (benediktms) [inline app/views/pages/home.html.erb:15]: move to navbar

File: app/views/pages/home.html.erb [hunk 4]
```diff
@@ -8,7 +8,12 @@
     <div class="col">
 
       <% if user_signed_in? %>
-        <p>Welcome back</p>
+      <%# Bruno added 4 lines below%>
+        <p>Welcome back [insert user name] </p>
+      <%= link_to 'Click here to view all our spaces and make a new booking', spaces_path %>
```
Comment (benediktms) [inline app/views/pages/home.html.erb:13]: remove

File: app/views/spaces/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<h2>Please find below all our spaces</h2>
```
Comment (benediktms) [inline app/views/spaces/index.html.erb:1]: unstage as nikki is working on this

File: app/views/spaces/show.html.erb [hunk 6]
```diff
@@ -1,12 +1,15 @@
```
Comment (benediktms) [inline app/views/spaces/show.html.erb:1]: remove this page from PR

File: db/seeds.rb [hunk 7]
```diff
@@ -5,3 +5,12 @@
 #
```
Comment (benediktms) [inline db/seeds.rb:5]: remove from PR

File: app/controllers/spaces_controller.rb (hunk 1 above)
Comment (brovani) [reply app/controllers/spaces_controller.rb]: done

File: app/views/pages/home.html.erb (hunk 3 above)
Comment (brovani) [reply app/views/pages/home.html.erb]: I deleted the file so will create a new branch with the changed navbar

File: app/views/spaces/index.html.erb (hunk 5 above)
Comment (brovani) [reply app/views/spaces/index.html.erb]: done

File: app/views/pages/home.html.erb (hunk 4 above)
Comment (brovani) [reply app/views/pages/home.html.erb]: deleted the file.

File: app/views/spaces/show.html.erb (hunk 6 above)
Comment (brovani) [reply app/views/spaces/show.html.erb]: done

**Pull request merged**